pycbc/waveform/generator.py

```diff
@@ -277,10 +293,14 @@ def generate(self, *args):
         rfparams = dict([(param, self.current_params[param])
             for param in self.rframe_generator.variable_args])
         hp, hc = self.rframe_generator.generate_from_kwargs(**rfparams)
+        hp._epoch = hc._epoch = self._epoch
```

Comment:
Looks like this is assigned to Alex now but will this change the meaning of ``tc``?

Before the ``_epoch`` for ``hp`` and ``hc`` was 0 right? So ``tc`` was essentially the ``dt`` value in ``apply_fd_time_shift`` (ie. seconds from the start of the data that was read).

Now it looks like it will need be the GPS time of the event right since ``_epoch`` is set to the GPS start time of the data read?